vela-templates/web-service.cue

```diff
@@ -0,0 +1,27 @@
+#Template: {
```

Comment:
Let's change the file name later to `webservice.cue` as I notice you changed the corresponding workload name above:)

Btw, It's great to splitting cue section from WorkloadDefintion/TraitDefinition. But I don't find cue file importing in  manifests.

Reply:
yes, not imported yet, will leave to another PR, hope to find some other guy can help.